### ISSUE TITLE
Manual shibboleth account linking does not redirect to initiator URL

### ISSUE THREAD

**Issue by michaelryanmcneill** [michaelryanmcneill/shibboleth#25]
This code doesn't seem to function properly (https://github.com/michaelryanmcneill/shibboleth/blob/master/options-user.php#L230-L232):

```php
$initator_url = shibboleth_session_initiator_url( get_edit_user_link() . '?shibboleth=link' );
wp_redirect( $initiator_url );
exit;
```

Instead, the page returns blank and never gets redirected to the indicator URL.

**Comment by michaelryanmcneill** [comment]
Yeah...it's a typo. Just took me looking closer at the code to realize. Working to fix now.

**Comment by michaelryanmcneill** [comment]
Fixed by afc5c5d.
